### DISCUSSION TITLE
How to Hide property from JSON but be able to still save to database via cast?

### DISCUSSION THREAD

**Discussion by linaspasv** [spatie/laravel-data#754] [Q&A]
Hello guys, 

I have found Hidden attribute but it looks to work differently than I have originally though. Is there another way or method to be able to hide some properties from JSON output but allow to save it to the database via cast?

**Comment by edalzell** [comment]
Ya I'm looking for the same thing. I need a few fields when the user is creating things, but want to save something a bit different. In my case I need some temefileupload stuff when data is being created, but when saved to the DB I only want to store a model id
